src/main_menu.cpp

```diff
@@ -789,6 +792,19 @@ bool game::opening_screen()
                     sel1 = 1;
                     layer = 2;
                     print_menu(w_open, sel1, iMenuOffsetX, iMenuOffsetY);
+                } else if (!templates.empty() && action == "DELETE_TEMPLATE") {
+                    if (query_yn(_("Are you sure you want to delete %s?"),
+                                 templates[sel3].c_str())) {
+                        const auto path = FILENAMES["templatedir"] + templates[sel3] + ".template";
+                        if (std::remove(path.c_str()) != 0) {
+                            popup(_("Sorry, something went wrong."));
+                        } else {
+                            templates.erase(templates.begin() + sel3);
+                            if (sel3 > templates.size() - 1) {
```

Comment:
Clang is being picky about this comparison, the simplest fix is casting sel3 to size_t.